assess/events/events.py

```diff
@@ -62,6 +62,17 @@ def events_from_process(cls, process):
         process_exit_dict["value"] = process_exit_dict["tme"] - \
             process_exit_dict["start_tme"]
 
+        if process_dict.get("pid", None) is None:
+            process_dict["pid"] = process_dict.get("node_id", None)
+            process_exit_dict["pid"] = process_dict.get("node_id", None)
+            if process_dict.get("_parent", None) is not None:
```

Comment:
Again, in case you do *not* have ``None`` values:
```suggestion
            if "_parent" in process_dict:
```

Reply:
In this case, `_parent` can actually be `None`, so this is intentional.